src/test/java/com/google/api/codegen/testdata/discoveries/java/java_appengine.v1beta5.json.baseline

```diff
@@ -8,7 +8,8 @@
  *    https://github.com/google/google-api-java-client.
  *    On other build systems, you can add the jar files to your project from
  *    https://developers.google.com/resources/api-libraries/download/appengine/v1beta5/java
- * 3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+ * 3. This sample uses Application Default Credentials for Auth.
```

Comment:
It does, but I think wrapping that is a mistake. It makes it more difficult for users to copy-paste the URL. If this is a real problem, I think the fix should be at the webpage, not the sample.